suites/quincy/rbd/tier-2_rbd_group.yaml

```diff
@@ -0,0 +1,101 @@
+# Tier2: Usecase to cover RBD grouo creation + immage add + io
+# Tier-level: 2
+# Test-Suite: tier-2_rbd_group_image_add_usecase.yaml
+#
+# Ceph RBD component.
+#
+# Cluster Configuration:
+#    Conf file - Baremetal config
+#    Node 4 must to be a client node
+tests:
+
+  # Setup the cluster
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-monitoring-stack: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+      desc: RHCS cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node4
+        install_packages:
+          - ceph-common
+          - fio
+        copy_admin_keyring: true
+      desc: Configure client node
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      desc: Install rbd-nbd and remove any epel packages
+      module: exec.py
+      name: Install rbd-nbd
+      config:
+        sudo: true
+        commands:
+          - "rm -rf /etc/yum.repos.d/epel*"
+          - "dnf install rbd-nbd -y"
+
+  - test:
```

Comment:
if we are not going to add any further test related to `group` feature
it's good to cover this test here itself
https://github.com/red-hat-storage/cephci/blob/master/suites/reef/rbd/tier-2_rbd_regression.yaml

Reply:
no no, as we discussed we are planning to only 2 usecases, not all functional combinations. So let this be there for 2 customer usecases. SInce not all customers may not do group snapshot I added "group image add" and "group snap" to loosely decouple
1. group image add 
2. group snap